scripts/Create-StorageContainer.ps1

```diff
@@ -71,59 +75,91 @@ trap [Exception]
     # this should trigger and catch and resume
     throw "Script execution failed: $($_)";
 }
+function New-Containers
+{
+    param(
+            [Parameter(Mandatory=$true)][string]$ContainerNames
+         )
+    # Create the container
+    [array]$storageContainerList = $ContainerNames.Split(",");
+    if ($storageContainerList.Length -gt 0)
+    {
+        Log-Message "Creating Multiple Storage Containers (Cli: $AzureCliVersion): $($ContainerNames)" -Context "Create Storage Containers"
+    }
 
-#########################
-#
-# ENTRY POINT
-#
-#########################
+    foreach($storageContainerName in $storageContainerList)
+    {
+        # todo: add retries for better resiliency
+        Log-Message "Creating Storage Container (Cli: $AzureCliVersion): $($storageContainerName)" -Context "Create Storage Containers"
 
-$invocation = (Get-Variable MyInvocation).Value 
-$currentPath = Split-Path $invocation.MyCommand.Path 
-Import-Module "$($currentPath)/Common.ps1" -Force
+        # Check if container already exists
+        $status = Get-ContainerExists -Container $storageContainerName
```

Comment:
- [ ] You have to be mindful of the powershell function naming. Ideally, on a best effort basis, you want {verb}-{target-noun} pattern. So something like: Get-StorageContainerStatus would be more appropriate